src/api/swagger.yaml

```diff
@@ -3262,16 +3274,31 @@ definitions:
       cpu_cores:
         type: integer
         description: Required number of CPU cores
-      ram_gib:
+      ram_mib:
```

Comment:
This most likely requires update in the UI code

Reply:
This is new addition to the API, so far not used in the UI.